easybuild/easyconfigs/g/Grace/Grace-5.1.25-intel-2019b-5build1.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'Grace'
+version = '5.1.25'
+versionsuffix = '-5build1'
+
+homepage = 'http://plasma-gate.weizmann.ac.il/Grace/'
+description = """Grace is a WYSIWYG tool to make two-dimensional plots of numerical data."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+source_urls = ['ftp://plasma-gate.weizmann.ac.il/pub/grace/src/grace5/']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['%(name)s-%(version)s%(versionsuffix)s.patch']
+checksums = [
+    '751ab9917ed0f6232073c193aba74046037e185d73b77bab0f5af3e3ff1da2ac',  # grace-5.1.25.tar.gz
+    '485fe80ef992e2068f5aa4bc94c0968b5bbef5491ff7c60c56152b61dcacbd46',  # Grace-5.1.25-5build1.patch
+]
+
+dependencies = [
+    ('motif', '2.3.8'),
+    ('zlib', '1.2.11'),
+    ('FFTW', '3.3.8'),
+    ('netCDF', '4.7.1'),
```

Comment:
@migueldiascosta Why is the list of dependencies different here compared to the `foss` toolchain?

Wouldn't it be better to align them?